### ISSUE TITLE
Can't register new volume type for etcd ebs

### ISSUE THREAD

**Issue by semoac** [kubernetes/kops#9141]
**1. What `kops` version are you running? The command `kops version`, will display
 this information.**

`Version 1.16.0 (git-4b0e62b82)`

**2. What Kubernetes version are you running? `kubectl version` will print the
 version if a cluster is running or provide the Kubernetes version specified as
 a `kops` flag.**

`v1.16.9`

**3. What cloud provider are you using?**

`AWS`

**4. What commands did you run?  What is the simplest way to reproduce this issue?**

* Change etcd-main ebs volume type from gp2 to io1 on AWS console.
* Edit cluster with kops edit cluster.
* Add `VolumeType` and `VolumeIops`

**5. What happened after the commands executed?**

```
A copy of your changes has been stored to "/var/folders/qv/kf14rx010h92hfryj8_0nl8m0000gn/T/kops-1.16.0-edit-ngzieyaml"

[spec.etcdClusters[main].Members[a].VolumeType: Forbidden: VolumeType cannot be changed, spec.etcdClusters[main].Members[a].VolumeIops: Forbidden: VolumeIops cannot be changed, spec.etcdClusters[main].Members[b].VolumeType: Forbidden: VolumeType cannot be changed, spec.etcdClusters[main].Members[b].VolumeIops: Forbidden: VolumeIops cannot be changed, spec.etcdClusters[main].Members[c].VolumeType: Forbidden: VolumeType cannot be changed, spec.etcdClusters[main].Members[c].VolumeIops: Forbidden: VolumeIops cannot be changed]
```

**6. What did you expect to happen?**

Kops should accept the new spec so `kops update cluster` shows that no changes were detected.

**7. Please provide your cluster manifest. Execute

```
apiVersion: kops.k8s.io/v1alpha2
kind: Cluster
metadata:
  creationTimestamp: "2018-04-24T14:58:47Z"
  generation: 3
  name: cs-main.k8s.local
spec:
  additionalPolicies:
    master: |
      [
        {
            "Effect": "Allow",
            "Action": [
                "autoscaling:DescribeAutoScalingGroups",
                "autoscaling:DescribeAutoScalingInstances",
                "autoscaling:DescribeTags",
                "autoscaling:SetDesiredCapacity",
                "autoscaling:TerminateInstanceInAutoScalingGroup"
            ],
            "Resource": "*"
        },
        {
            "Effect": "Allow",
            "Action": [
                "s3:*"
            ],
            "Resource": [
                "arn:aws:s3:::main-kubernetes-etcd-backup/cs-main.k8s.local/*"
            ]
        },
        {
            "Effect": "Allow",
            "Action": [
                "ec2:Describe*",
                "ec2:Get*"
            ],
            "Resource": [
                "*"
            ]
        }
      ]
    node: |
      [
        {
          "Sid": "K8sRoute53List",
          "Effect": "Allow",
          "Action": [
              "route53:ListHostedZones",
              "route53:ListResourceRecordSets"
          ],
          "Resource": [
              "*"
          ]
        },
        {
          "Sid": "K8sRoute53Change",
          "Effect": "Allow",
          "Action": [
              "route53:ChangeResourceRecordSets"
          ],
          "Resource": [
              "arn:aws:route53:::hostedzone/*"
          ]
        }
      ]
  api:
    loadBalancer:
      type: Public
  authentication:
    aws: {}
  authorization:
    rbac: {}
  channel: stable
  cloudProvider: aws
  configBase: s3://main-kubernetes-state-store/cs-main.k8s.local
  etcdClusters:
  - etcdMembers:
    - instanceGroup: master-us-east-1a
      name: a
    - instanceGroup: master-us-east-1b
      name: b
    - instanceGroup: master-us-east-1c
      name: c
    name: main
    provider: Manager
    version: 3.2.18
  - etcdMembers:
    - instanceGroup: master-us-east-1a
      name: a
    - instanceGroup: master-us-east-1b
      name: b
    - instanceGroup: master-us-east-1c
      name: c
    name: events
    provider: Manager
    version: 3.2.18
  iam:
    allowContainerRegistry: true
    legacy: false
  kubeDNS:
    provider: CoreDNS
  kubelet:
    allowedUnsafeSysctls:
    - net.core.somaxconn
    - net.ipv4.tcp_keepalive_time
    - net.ipv4.tcp_keepalive_probes
    - net.ipv4.tcp_keepalive_intvl
    - net.ipv4.tcp_fin_timeout
    - net.ipv4.tcp_keepalive_probes
    anonymousAuth: false
    authenticationTokenWebhook: true
    authorizationMode: Webhook
    cpuCFSQuota: true
    cpuCFSQuotaPeriod: 10ms
    enforceNodeAllocatable: pods
    featureGates:
      CustomCPUCFSQuotaPeriod: "true"
    kubeReserved:
      cpu: 100m
      memory: 200Mi
    systemReserved:
      cpu: 100m
      memory: 300Mi
  kubernetesApiAccess:
  - 0.0.0.0/0
  kubernetesVersion: 1.16.9
  masterInternalName: api.internal.cs-main.k8s.local
  masterPublicName: api.cs-main.k8s.local
  networkCIDR: 172.29.0.0/16
  networking:
    calico:
      crossSubnet: true
      majorVersion: v3
      prometheusMetricsEnabled: true
      typhaReplicas: 3
  nonMasqueradeCIDR: 100.64.0.0/10
  sshAccess:
  - 172.29.0.0/16
  subnets:
  - cidr: 172.29.32.0/19
    name: us-east-1a
    type: Public
    zone: us-east-1a
  - cidr: 172.29.64.0/19
    name: us-east-1b
    type: Public
    zone: us-east-1b
  - cidr: 172.29.96.0/19
    name: us-east-1c
    type: Public
    zone: us-east-1c
  topology:
    dns:
      type: Public
    masters: public
    nodes: public
```

**8. Please run the commands with most verbose logging by adding the `-v 10` flag.
  Paste the logs into this report, or in a gist and provide the gist link here.**

```
I0518 23:43:30.574430   14250 status.go:68] Listing EC2 Volumes
I0518 23:43:30.575480   14250 request_logger.go:45] AWS request: ec2/DescribeVolumes
I0518 23:43:30.984319   14250 status.go:40] Cluster status (from cloud): {"etcdClusters":[{"name":"main","etcdMembers":[{"name":"c","volumeId":"vol-0da5a09f2c8463964"},{"name":"b","volumeId":"vol-06b08aed38a5506d4"},{"name":"a","volumeId":"vol-00c3de42b4fe8f5e0"}]},{"name":"events","etcdMembers":[{"name":"c","volumeId":"vol-02bf1acee88f76940"},{"name":"b","volumeId":"vol-0e16f9f09c676c1be"},{"name":"a","volumeId":"vol-0bc83f24415cc4a71"}]}]}
I0518 23:43:30.984367   14250 s3fs.go:220] Reading file "s3://main-kubernetes-state-store/cs-main.k8s.local/config"
A copy of your changes has been stored to "/var/folders/qv/kf14rx010h92hfryj8_0nl8m0000gn/T/kops-1.16.0-edit-e0133yaml"

[spec.etcdClusters[main].Members[a].VolumeType: Forbidden: VolumeType cannot be changed, spec.etcdClusters[main].Members[a].VolumeIops: Forbidden: VolumeIops cannot be changed, spec.etcdClusters[main].Members[b].VolumeType: Forbidden: VolumeType cannot be changed, spec.etcdClusters[main].Members[b].VolumeIops: Forbidden: VolumeIops cannot be changed, spec.etcdClusters[main].Members[c].VolumeType: Forbidden: VolumeType cannot be changed, spec.etcdClusters[main].Members[c].VolumeIops: Forbidden: VolumeIops cannot be changed]
```

**9. Anything else do we need to know?**

My original volumes were having performance issues. Changing the volume type was necessary.

I need to understand what Kops will do when I try to run a `kops update cluster`. It will try to revert the changes? How can I register the new state inside the kops spec?

**Comment by semoac** [comment]
I edited the bucket directly.

**Comment by MoShitrit** [comment]
I have come across the same issue yesterday and followed the same steps (modified the volume directly on AWS EBS and then edited the bucket directly to reflect the changes).
Following discussion in office hours of August 14, 2020, kops should probably allow this type of change and support it, since EBS does as well. This is probably something that was done for legacy reasons from back when EBS didn't support it but now that it's supported- might be worth looking into allowing this.
/reopen

**Comment by olemarkus** [comment]
This was fixed in #10438 
/remove-lifecycle rotten
